src/client/src/translations/whitelist_en.json

```diff
@@ -96,11 +96,7 @@
   "privacyModal.beingRedirected",
   "privacyModal.privacyStatement",
   "privacyModal.OK",
-  "projectName.projectNameTitle",
-  "projectName.projectName",
   "projectName.ariaProjectName",
-  "projectName.outputPathTitle",
```

Comment:
seems like this is still being used..

Reply:
Sorry, this was automatically removed - just added it back for both